fix(contrib/gce): updated GCE docs with new cloud dns config and commas

tscheepers opened the pull request:
Resolves:
```
WARNING: Lists should be separated by commas, try "--metadata-from-file user-data=gce-user-data,sshKeys=/Users/Thijs/.ssh/deis.pub". If you intend to use sshKeys=/Users/Thijs/.ssh/deis.pub for positional arguments, put them before the flag.
WARNING: Lists should be separated by commas, try "--disk name=cored3,device-name=coredocker". If you intend to use device-name=coredocker for positional arguments, put them before the flag.
```

deis-admin commented:
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

mboersma commented:
Code LGTM. Thanks for finding and fixing this @tscheepers!

tscheepers commented:
Hi @mboersma, I've updated the commit message, added a couple more commas and I added new Google Cloud DNS config as well. With these changes the docs are up to date again.

carmstrong commented:
Code LGTM.

mboersma commented:
I'll test this manually today now that I sorted out my GCE billing account. :money_with_wings:

mboersma commented on docs/installing_deis/gce.rst:
```diff
@@ -115,8 +115,8 @@ Launch 3 instances. You can choose another starting CoreOS image from the listin
       --machine-type n1-standard-2 \
-      --metadata-from-file user-data=gce-user-data sshKeys=~/.ssh/deis.pub \
-      --disk name=cored${num} device-name=coredocker \
+      --metadata-from-file user-data=gce-user-data,sshKeys=~/.ssh/deis.pub \
```
This change breaks instance creation because the tilde `~` is no longer expanded (don't ask me why...):
```
ERROR: (gcloud.compute.instances.create) Could not read metadata key [sshKeys] from file [~/.ssh/deis.pub]: No such file or directory
```
Removing the space causes the tilde to be interpreted again, but emits the warning:
```
WARNING: We noticed that you are using space-separated lists, which are deprecated. Please transition to using comma-separated lists instead (try "--metadata-from-file user-data=gce-user-data,sshKeys=/Users/matt/.ssh/deis.pub"). If you intend to use [sshKeys=/Users/matt/.ssh/deis.pub] as positional arguments, put the flags at the end.
Created [https://www.googleapis.com/compute/v1/projects/deis-testing-0001/zones/us-central1-c/instances/core1].
NAME  ZONE          MACHINE_TYPE  PREEMPTIBLE INTERNAL_IP  EXTERNAL_IP     STATUS
core1 us-central1-c n1-standard-2             10.240.11.1 107.178.201.123 RUNNING
```
@tscheepers could you replace the tilde with `$HOME`? That makes everything happy:
```
  --metadata-from-file user-data=gce-user-data,sshKeys=$HOME/.ssh/deis.pub \
```

mboersma commented:
Code LGTM.
